artiq/test/test_pc_rpc.py

```diff
@@ -92,6 +92,31 @@ def test_asyncio_echo(self):
     def test_asyncio_echo_autotarget(self):
         self._run_server_and_test(self._loop_asyncio_echo, pc_rpc.AutoTarget)
 
+    def test_rpc_encode_function(self):
+        """Test that `pc_rpc` can encode a function properly.
+
+        Used in `get_rpc_method_list` part of
+        :meth:`artiq.protocols.pc_rpc.Server._process_action`
+        """
+
+        def _annotated_function(
+            arg1: str, arg2: np.ndarray = np.array([1,])
+        ) -> np.ndarray:
+            """Sample docstring."""
+            return arg1
+
+        argspec, docstring = pc_rpc.Server._document_function(_annotated_function)
+        print(argspec)  # for debug
```

Comment:
Prints in tests are generally "for debug", this comment doesn't provide much info.